specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2019-08-08/storageaccounts.json

```diff
@@ -33,35 +33,38 @@
             "$ref": "storage.json#/parameters/SubscriptionIdParameter"
           },
           {
-            "$ref": "storage.json#/parameters/ResourceGroupNameParameter"
+            "$ref": "storage.json#/parameters/LocationParameter"
           },
           {
-            "$ref": "storage.json#/parameters/FarmIdParameter"
+            "$ref": "storage.json#/parameters/ApiVersionParameter"
           },
           {
-            "$ref": "storage.json#/parameters/ApiVersionParameter"
+            "$ref": "#/parameters/FilterParameter"
```

Comment:
Is this an ODATA endpoint?

Reply:
https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-odata

Reply:
fixed